instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -255,11 +256,13 @@ def _get_operation_name(handler, request):
 def _start_span(tracer, handler, start_time) -> _TraceContext:
     token = context.attach(extract(handler.request.headers))
```

Comment:
wouldn't this always extract tracing information from carrier and set it in the active context resulting in the remote context always being the parent?

Reply:
I did not realize that would happen. I will make edits so that I only extract context from the carrier if a span is not present in the current context.